Tipstaff/Areas/Admin/Views/Solicitors/Index.cshtml

```diff
@@ -95,7 +95,7 @@
             @Html.DisplayFor(modelItem => item.active)
         </td>
         <td>
-            @item.TipstaffRecords.Count()
+           / @item.TipstaffRecords.Count()
```

Comment:
what is that slash? is it a typo or a display/layout thing?

Reply:
typo
..will update